Tensile/Source/lib/CMakeLists.txt

```diff
@@ -73,14 +73,6 @@ if(TENSILE_USE_HIP)
         )
 endif()
 
-if(TENSILE_USE_OPENCL)
```

Comment:
I found one more "if(TENSILE_USE_OPENCL)" in this file.
Better to remove it as well.